homeassistant/components/alexa/handlers.py

```diff
@@ -28,6 +27,7 @@
     valve,
     water_heater,
 )
+from homeassistant.components.alarm_control_panel import AlarmControlPanelState
```

Comment:
For other platforms we import the whole platform. To be consequent and for readability, I suggest we keep the new code aligned with that of the other platforms.

Reply:
Done!